perma_web/perma/views/common.py

```diff
@@ -210,6 +210,16 @@ def make_warcinfo(filename, guid, coll_title, coll_desc, rec_title, pages):
         if (not capture or capture.status != 'success') and link.screenshot_capture and link.screenshot_capture.status == 'success':
             return HttpResponseRedirect(reverse('single_linky', args=[guid])+"?type=image")
 
+    # Special handling for mobile pdf viewing because it can be buggy
+    # Redirecting to a download page if on mobile
+    try:
+        capture_mime_type = capture.mime_type()
+        redirect_to_download_view = redirect_to_download(capture_mime_type, raw_user_agent)
+    except AttributeError:
+        # if capture is deleted, we will catch it here
```

Comment:
Really liked your commit message here. "If capture is deleted, then mime type does not exist. Catch error." Total nitpick, totally up to you: do you think that might be a clearer comment here than the above?